### DISCUSSION TITLE
I am confused as to how to use AnyType

### DISCUSSION THREAD

**Discussion by inikishev** [anyproto/contributors#139] [Q&A]
I watched the video and it looked cool... However I feel dumb because I can't for the life of me find how to create a simple note. Apparently I can create a "collection" and a "view", and then I went to favourites and created a new "object", and the "objects" I created appear into collection view. Which I am sure is a super powerful thing. But I've been reading the docs and watching the videos but I can't find it how to actually make a note in AnyType. Also I may or may not have pressed all buttons in the app in hopes that one of them will create a note and it may have broken it.

**Comment by inikishev** [comment]
Update, I have found a "notes"... something in my recycling bin. Turns out it is a set and I can actually add new notes in it. So now I have the only question which is how to get a tree structure like on the video? I found that when I press "add widget" it lets me create a widget of a type "tree". However for some reason there is no "tree" view for my notes set, and I couldn't figure out how to make it appear. So my question is, how to get the tree view like on the video "[Anytype Beta Demo - 2023](https://www.youtube.com/watch?v=dh_3NHY5eVs)". Oh and also when I created a "tree" widget from one of the things that it let me do it on, I also couldn't actually make a branch, it would still look like a list no matter what button I pressed

**Comment by filip-anytype** [comment]
For anything that you don't know, you can check our [Docs](https://doc.anytype.io/anytype-docs/). Check this for [Objects](https://doc.anytype.io/anytype-docs/basics/object-editor), and this specifically for [creating new objects](https://doc.anytype.io/anytype-docs/basics/object-editor/create-an-object).
You can't use tree widgets with sets / collections (and probably some other types). Tree widgets are only used with "regular" objects like pages.
If you add links inside the object you used for the widget, those links will appear in the widget. If you then add links inside the objects you already linked in the original object, those will show up nested in the widget (hence the tree widget).
Hopefully this explains everything.
